ReSharper.FSharp/src/FSharp.Psi.Features/src/Parsing/FSharpImplTreeBuilder.fs

```diff
@@ -159,8 +158,7 @@ type FSharpImplTreeBuilder(lexer, document, decls, lifetime, projectedOffset, li
 
                 | _ ->
 
-                for m in members do
-                    x.ProcessTypeMember m
+                x.ProcessTypeMembers(members)
```

Comment:
It should check the primary constructor separately, as it comes before `=` and functional types representations.